pom.xml

```diff
@@ -112,6 +112,7 @@
         <dependency.jakarta-ee-json-impl.version>1.1.6</dependency.jakarta-ee-json-impl.version>
         <dependency.jakarta-json-path.version>2.9.0</dependency.jakarta-json-path.version>
         <dependency.json-smart.version>2.5.1</dependency.json-smart.version>
+        <dependency.org.testng.version>7.9.0</dependency.org.testng.version>
```

Comment:
Let's refactor this to just be `dependency.testng.version` if possible. We don't usually specify the groupId segments in the version properties here.

Reply:
Also, any reason why we're using `7.9.0` specifically instead of the latest `7.10.2`?

Reply:
7.9.0 was last version of testNG released before current version of report-portal testng-agent. Its not working with a newer version